gspread-project/push_data.py

```diff
@@ -18,7 +25,7 @@ def create_spreadsheet(gc: gspread.client, sheet_title: str) -> gspread.spreadsh
     this function create a new spreadsheet
     """
     sheet = gc.create(sheet_title)
-    print("google sheet created succesfully")
```

Comment:
Print() is needed here; you've already used logging.info(). You may use print() only when debugging.